elements/pf-dropdown/pf-dropdown.ts

```diff
@@ -65,7 +62,7 @@ export class PfDropdown extends LitElement {
   static override readonly shadowRootOptions: ShadowRootInit = { ...LitElement.shadowRootOptions, delegatesFocus: true };
 
   /**
-   * Disable the dropdown trigger element
+   * Disable the dropdown
```

Comment:
```suggestion
   * When disabled, the dropdown can still be toggled open and closed via keyboard, but menu items cannot be activated.
```

The [upstream description for `isDisabled` prop](http://v4-archive.patternfly.org/v4/components/dropdown/#dropdowntoggle) is not particularly relevant to us, so I decided to wing it. How does this read to you?